lib/tether/tether.go

```diff
@@ -42,11 +44,16 @@ import (
 )
 
 const (
-	// The maximum number of records to keep for restarting processes
+	// MaxDeathRecords The maximum number of records to keep for restarting processes
 	MaxDeathRecords = 5
 
 	// the length of a truncated ID for use as hostname
 	shortLen = 12
+
+	// temp directory to copy existing data to mounts
+	bindDir = "/.bind"
```

Comment:
We don't have a label for nfs mounts. We could use a combination of hostname and path replacing `/` with `-`.